test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -747,6 +755,83 @@ def __are_requirements_matching(param_dic=None, test=None):
             if not eff_tc_req['no_comm_on_mom']:
                 return False
 
+    def hardware_report(self):
+        """
+        function used to get hardware report
+        every 5 minutes
+        """
+        du = DshUtils()
+        self.hardware_report_timer = Timer(300, self.hardware_report)
+        self.hardware_report_timer.start()
+        self.hardware_monitor_start = True
```

Comment:
remove all reference of `self.hardware_monitor_start` from this file as you are not using it anymore.